deeptime/markov/_observables.py

```diff
@@ -0,0 +1,66 @@
+from typing import Union
+
+import numpy as np
+
+from ..base import Observable
+
+
+class MembershipsObservable(Observable):
+
+    @staticmethod
+    def _to_markov_model(model):
+        if hasattr(model, 'prior'):
+            model = model.prior
+        if hasattr(model, 'transition_model'):
+            model = model.transition_model
+        from deeptime.markov.msm import MarkovStateModel
+        assert isinstance(model, MarkovStateModel), f"This should be a Markov state model but was {type(model)}."
+        return model
+
+    def __init__(self, test_model, memberships,
```

Comment:
`memberships: Union[PCCAModel, np.ndarray]` ? Will otherwise fail below when calling `shape`

Reply:
right! i cannot really do it that way because of circular imports (:exploding_head:) but adding a docstring